src/Symfony/Component/Finder/Tests/Iterator/SizeRangeFilterIteratorTest.php

```diff
@@ -41,7 +41,7 @@ public function getAcceptData()
         ];
 
         return [
-            [[new NumberComparator('< 1K'), new NumberComparator('> 0.5K')], $this->toAbsolute($lessThan1KGreaterThan05K)],
+            [[new NumberComparator('< 1K'), new NumberComparator('> 0.5K')], self::toAbsolute($lessThan1KGreaterThan05K)],
```

Comment:
Method was already declared `static`

Reply:
`getAcceptData()` is not `static`.

Reply:
Please check the PR header, those gets all fixed automatically via rector, these are just things were we have to add manuall work

Reply:
Oh, I see. Would it make sense to push these manual changes as a separate commit to #48668? Other than that, this PR is fine as far as I can tell.

Reply:
I want to bring the manual stuff beside this PR into the codebase, unmerge everything and then apply the automatic changes branch by branch.